CHANGES.md

```diff
@@ -0,0 +1,36 @@
+## 2.0 (2020-05-06)
```

Comment:
I think dune-release tries to use the first heading as the name for which tag to create. We can either keep this as `## 2.0` or manually specify the tag to use via the cli `dune-release <version name>` when preparing the release